truss/util/path.py

```diff
@@ -75,3 +96,65 @@ def build_truss_shadow_target_directory(stub: str, truss_path: Path) -> Path:
     target_directory_path = Path(Path.home(), ".truss", "models", f"{stub}-{suffix}")
     target_directory_path.mkdir(parents=True, exist_ok=True)
     return target_directory_path
+
+
+def load_trussignore_patterns(truss_ignore_file: Path):
+    """Load patterns from a .truss_ignore file"""
+    patterns = []
+    with truss_ignore_file.open() as f:
```

Comment:
nit: I think it might be cleaner to `f.readlines()` instead of iterating then filtering there.